docs/events/2021_timesketch_summit.md

```diff
@@ -0,0 +1,53 @@
+# Timesketch Summit 2021
+
+On **March 10th 2021 16:00 - 22:00 UTC** the first Timesketch summit will
+take place.
+
+## About Timesketch
+
+Timesketch is an open source tool for collaborative forensic timeline analysis.
+Using sketches you and your collaborators can easily organize your timelines and
+analyze them all at the same time. Add meaning to your raw data with rich
+annotations, comments, tags and stars.
+
+## Agenda and talks
+
+The agenda will be posted on
+[Github](https://github.com/google/timesketch/tree/master/docs/events/2021_timesketch_summit.md)
+end of February.
+
+- Presentations: 16:00 - 19:00 UTC
+- Workshops: 19:00 - 22:00 UTC
+
+## Where
+
+The Timesketch summit will take place virtually.
+
+## Who can participate
+
+Everyone who is interested in Timesketch is welcome. **The event is free**.
+
+## Registration
+
+Please use the [Google form](https://forms.gle/1D23n4SkoCPay1eDA) to sign up.
+
+## Call for Presentations & Workshops
+
+We invite Timesketch users and developers to submit a presentation or a workshop
+topic. Anything related to Timesketch is welcome, such as analysis techniques,
+deployment and development to name a few.
+
+We are looking for short format talks 20 min + 10 min Q&A. Workshops will have
+a 3 hour time slot.
+
+Please use the [Google form](https://forms.gle/1D23n4SkoCPay1eDA) to suggest
+your contribution to the summit. The call for presentations and workshops will
+run till **Feb 25th**.
+
+If you have ideas outside of the suggested formats, feel free to reach out.
+
+## Questions
+
+If you have any questions, please reach out to the team via 
+[slack in the Timesketch channel](https://github.com/open-source-dfir/slack) or
+via [Twitter](https://twitter.com/alexanderjaeger)
```

Comment:
Let's skip personal Twitter accounts here so we don't spread the comms too much. Twitter will be the main outreach anyways and folks who wanna ask stuff there can just do it on the thread. :) WDYT?

I opt to use the Slack channel as the main communication channel, that way we also get the interested folks signed up and ready for the summit.

Please add the link to the self registration: http://join-open-source-dfir-slack.herokuapp.com/

Reply:
agree

Reply:
ACK, removed the twitter link, the self registration and description is in the linked github repo, that should be enough imho